examples/fit-transform-tfidf.py

```diff
@@ -0,0 +1,113 @@
+import functools
+
+import dask
+import pandas as pd
+import pyarrow as pa
+import requests
+import toolz
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+import xorq as xo
+import xorq.expr.udf as udf
+import xorq.vendor.ibis.expr.datatypes as dt
+from xorq.common.caching import ParquetCacheStorage
+from xorq.expr.udf import make_pandas_expr_udf, wrap_model
+
+
+@functools.cache
+def hackernews_stories(n=1000):
```

Comment:
Does it make sense to use parquet or even CSV instead of pickle? I think parquet/CSV would be a lot more relatable for data things.

Reply:
the current solution uses a per-item_id json